frontend/src/App.css

```diff
@@ -1,12 +1,17 @@
+.fixedBottomBtn {
+    position: fixed;
+    border-radius: 0px;
+    bottom: 0px;
+    left: 0px;
+    right: 0px;
+  }
```

Comment:
Doesn't seem to be getting used anywhere. (It's for merchant screens if I remember correctly.) Consider removing it